### ISSUE TITLE
Confirm does not show dialogue box

### ISSUE THREAD

**Issue by cyrilbensonyu** [dillingham/nova-button#75]
`           Button::make('Archive', 'user-archive')
                ->style('danger-outline')
                ->confirm('Are you sure you want to archive this user?')
                ->reload(),
`

I'm using this line of code and confirm dialogue does not show up whenever I click the button. I'm using the latest Laravel Nova version (3.12).

**Comment by nilsE** [comment]
...same here. confim() works on the detail form, but not on index

**Comment by RomkaLTU** [comment]
It works but only on the last entry, seems JS bug.

**Comment by OthmanAlseagh** [comment]
@dillingham did you solve this issue 

i have the same

when i use confirm() , only the last button can be clickable

**Comment by dalmarcolucas** [comment]
same problem here, only the last button show the confirm dialog.

**Comment by piteurstudioislam** [comment]
did you solve the problem

**Comment by cyrilbensonyu** [comment]
@piteurstudioislam Yes, I forked this repo and created my own package just to fix the bug.

**Comment by piteurstudioislam** [comment]
can yopu give it to me

**Comment by piteurstudioislam** [comment]
@cyrilbensonyu  cAN YOU GIVE The package

**Comment by cyrilbensonyu** [comment]
sorry, I did not made it public

**Comment by piteurstudioislam** [comment]
Did you mad it now @cyrilbensonyu

**Comment by stardothosting** [comment]
@cyrilbensonyu would love it if you could share this package - this is an open source project and lots of people depend on it.

**Comment by stardothosting** [comment]
@piteurstudioislam The fix in another thread referencing a forked pull request worked for me : https://github.com/qikker-online/nova-button/commit/dc984708eecbaa887d4455828135912114cf8b41
